### DISCUSSION TITLE
Force logout or delete session of a user?

### DISCUSSION THREAD

**Discussion by fpolli** [laravel/framework#52460] [Q&A]
I have been looking for a way that a site admin can force logout a user on his site or delete the user's session to force a login, but I cannot find a solution other than storing sessions in a database and locating by user id field. Ok, I can do that, but I would like to be able to use RAM caching for sessions for the performance.

Is there any way to do this? I only see ways to do this for the logged in user of the current request.

**Comment by ghost** [comment]
First you need to know what sessions are used: If you use sticky session in memory(cache) or in db (for multi-container application) or If you use file sessions (for single container application). Then read the documentation about these.
See for example https://laravel.com/docs/8.x/authentication#invalidating-sessions-on-other-devices
